script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -613,7 +613,7 @@ You see, the memories and personality the host body develops as it grows %{i}are
 -- Ａが成人に達して吸血鬼になった時、覚醒したロアの<情報|たましい>がＡになるのではないんです。
 // なる as 'become' feels a bit wishy washy, and 'replace' seems pretty accurate esp with the anims backing it up
 Once Person A reaches adulthood they becomes a vampire,
- but it would be wrong to say that when Roa awakens his <information|s o u l     > %{i}replaces%{/i} Person A.
+ but it would be wrong to say that when Roa awakens, his <information|s o u l     > %{i}replaces%{/i} Person A.
```

Comment:
This one feels like an awkward break to me

Reply:
oh yeah, i didn't notice this line was continuing from L615, i agree to revert